map.py

```diff
@@ -25,8 +25,11 @@
 import shutil
 import urllib
 import random
+import gi
 
+gi.require_version('Gtk', '3.0')
 from gi.repository import Gtk
+gi.require_version('Gtk', '3.0')
```

Comment:
Yes, I tested it. As we discussed earlier in this pull request https://github.com/sugarlabs/getiabooks/pull/9 there were no warnings related to Gdk on my console. But as it is a good practice to specify versions before importing, I also specified Gdk version, which by mistake is replaced by Gtk, I am sorry about that.And since no warnings for Gdk, not able to figure out that mistake.
Thanks!

Reply:
Ah, yes, it is the unusual ordering of imports.  You might change to put the import of Gdk before the import of Gtk.  This would be more natural, given that Gtk imports Gdk itself.

Reply:
Done.Thanks!